anxcloud/resource_vlan_test.go

```diff
@@ -25,12 +25,21 @@ func TestAccAnxCloudVLAN(t *testing.T) {
 		CheckDestroy:      testAccCheckAnxCloudVLANDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCheckAnxCloudVLAN(resourceName, locationID, customerDescription, false),
```

Comment:
testing for `VMProvisioning=false` is not done in any test anymore, is this intentional?